transforms/code/code_quality/kfp_ray/Makefile

```diff
@@ -19,10 +23,14 @@ test-src::
 
 test-image::
 
+publish::
+
 image::
 
 load-image::
 
+set-versions: workflow-reconcile-requirements
```

Comment:
Does it make sense to rename `workflow-reconcile-requirements` to `set-versions` to reduce the number of rules?

Reply:
yes, but I didn't want to risk breaking anything that might have used the workflow target :).